docs/synapse/userguides/storm_ref_type_specific.rstorm

```diff
@@ -215,6 +223,96 @@ nodes where the ``ou:name`` is present in the ``:name`` property or as an elemen
+:ref:`storm-whitespace-literals`.
+  
+A ``duration`` can also be specified as the number of epoch milliseconds, enclosed in parentheses:
+  
+  ``(218262777)``
+
+.. TIP::
+  
+  Similar to :ref:`type-time` types, Synapse expects that users will generally enter duration values using
+  "human friendly" strings, and will attempt to parse the input as such. Using parentheses tells Synapse
+  to interpret the value as a raw integer.
+
+
+Insertion
++++++++++
+
+When setting a ``duration`` value, enter the value using either format above.
+
+**Examples:**
+
+Set the duration of a compromise (``risk:compromise`` node) to six days, six hours, 46 minutes, and
+23 seconds:
+
+.. storm-cli:: [ risk:compromise = * :name = 'example compromise' :duration = '6D 06:46:23' ]
+
+Or:
+
+.. storm-cli:: [ risk:compromise = * :name = 'example compromise' :duration = (542783000) ]
+
+Values can be set at any level of granularity; e.g., ``[ :duration = 19D ]`` is acceptable, as are
+values to milisecond resolution.
+
+.. NOTE::
+
+  Forms that have ``duration`` properties may have related :ref:`type-time` properties as appropriate
+  for the form, such as a start time or end time or both. A compromise (``risk:compromise`` node) is
+  one example and includes start time (``:time``), end time (``:lasttime``), and duration (``:duration``).
+  This allows you to set any or all properties, depending on the information available. For example,
+  you may know when a compromise was fully remediated (ended), but not when it began or how long it lasted.
+  
+  While start, end, and duration values are related, Synapse does **not** currently calculate values
+  for these properties. That is, entering a start time and a duration will not automatically calculate
+  and set the ending time. Similarly, if all three values are set and you modify one of the values,
+  the other value(s) are not updated to reflect the change (e.g., changing the duration will not
+  automatically update the ending time).
+  
+  That said, in cases where a node has start, end, and duration properties and you have two of the
+  three values, you can use variables_ to calculate and set the third value. For example, given a
+  ``risk:compromise:name = 'my compromise``  with ``:time = '2023/09/22 10:40'`` and
+  ``:lasttime = '2023/10/06 14:00'`` you could set the ``:duration`` value using the following Storm query:
+  
+  ``risk:compromise:name='my compromise' $min=:time $max=:lasttime [ :duration=( $max - $min ) ]``
+  
+  This will set ``:duration = 14D 03:20:00.000``.
+
+
+Operations
+++++++++++
+
+As ``duration`` types are stored as integers and support any operations suitable for those types. This
+includes:
+
+- comparison using mathematical operators (such as greater than ( ``>``));
+- finding low or high values using the :ref:`storm-min` or :ref:`storm-max` commands;
+- use of the range ( ``*range=`` ) comparison operator;
+- etc.
```

Comment:
`etc.` seems to be formatted as an item in the list, which looks different from how it is used in other lists.